source/_components/darksky.markdown

```diff
@@ -159,6 +159,10 @@ scan_interval:
   type: time
 {% endconfiguration %}
 
+<p class='note'>
+Please note that some monitored conditions, such as `temperature_high` or `temperature_high`, may only work when setting the `forecast` attribute to at least `0` (current day).
```

Comment:
`temperature_high` or `temperature_high`?

Reply:
Thanks. That obviously wasn't correct.